angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -76,6 +77,7 @@ import {
               <sprk-link
                 linkType="unstyled"
                 [href]="choice.href"
+                [analyticsString]="choice.analyticsString"
```

Comment:
This would be a minor version bump or a bug depending on how we view it. This component didnt have the ability to pass in an analytics string before, and now i added it. Could be viewed as a new feature or a bug (since we should of had it there before). What do you all think @RCopeland , @yodasw16 @13protons ?